extensions/src/main/java/com/redhat/lightblue/extensions/sequance/Sequance.java

```diff
@@ -0,0 +1,25 @@
+package com.redhat.lightblue.extensions.sequance;
+
+import com.redhat.lightblue.metadata.SimpleField;
+
+/**
+ * TODO: Does SimpleField have all the info required by the backend to manage sequences?
+ *
+ * @author mpatercz
+ *
+ */
+public interface Sequance {
+
+    public void createSequance(SimpleField field);
+
+    public void removeSequance(SimpleField field);
+
```

Comment:
It is easier to not rely on a createSequence call() before using the
sequence. Sequence generator can create the sequence if it is not there.

On Tue, Aug 18, 2015 at 2:52 AM, Marek <notifications@github.com> wrote:

> In
> extensions/src/main/java/com/redhat/lightblue/extensions/sequance/Sequance.java
> <https://github.com/lightblue-platform/lightblue-core/pull/431#discussion_r37276955>
> :
>
> > +package com.redhat.lightblue.extensions.sequance;
> > +
> > +import com.redhat.lightblue.metadata.SimpleField;
> > +
> > +/**
> > + * TODO: Does SimpleField have all the info required by the backend to manage sequences?
> > + *
> > + * @author mpatercz
> > + *
> > + */
> > +public interface Sequance {
> > +
> > +    public void createSequance(SimpleField field);
> > +
> > +    public void removeSequance(SimpleField field);
> > +
>
> It's for creating and removing sequences using api rather than via
> metadata. Could be useful for plugins, etc. Or not?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/431/files#r37276955>
> .
>